Templates/Swift/ResponseType.stencil

```diff
@@ -1,25 +1,30 @@
 {% extends "Base.stencil" %}
 {% block content %}
+{% if computedFragmentFields.count != 0 %}
+@dynamicMemberLookup
+
+{% endif %}
 {% if not asFile %}{{ accessLevel }} {% endif %}struct {{ name }}: GraphApiResponse, Equatable {
 	// MARK: - Response Fields
 	{% for field in fields %}
 		{% with field.attributes as attributes %}{% include "Helpers/Attributes.stencil" %}{% endwith %}
 		{{ accessLevel }} var {{ field|renderPropertyDeclaration }}
 	{% endfor %}
 
-	{% for field in computedFragmentFields %}
-		{% with field.attributes as attributes %}{% include "Helpers/Attributes.stencil" %}{% endwith %}
-		{{ accessLevel }} var {{ field|renderComputedPropertyDeclaration }} {
-			get {
-				return as{{ field.parentFragment.name }}Fragment{% if field.parentFragment.conditionallySelected %}?{% endif %}.{{ field.name }}
-			}
-			{% if not field.parentFragment.conditionallySelected %}
-			set {
-				as{{ field.parentFragment.name }}Fragment{% if field.parentFragment.conditionallySelected %}?{% endif %}.{{ field.name }} = newValue
-			}
-			{% endif %}
+	{% if computedFragmentFields.count != 0 %}
```

Comment:
Does computedFragmentFields contain the same fields that fragmentSpreads would contain? I kind of recall doing something special if you had multiple fragment spreads on a single type, but maybe I'm wrong. I'd test this with a type that has multiple fragment spreads on it.